app/controllers/link_check_rules_controller.rb

```diff
@@ -27,7 +27,11 @@ def new
   end
 
   def edit
-    @partners = Partner.where(:id.in => current_user.manage_partners).asc(:name)
+    @partners = if current_user.admin?
```

Comment:
ternary statement or a helper method might be nice for this one. But all 3 options are okay i just think they are abit cleaner